Support for SharedAccessSignature Tokens for Azure IO (#1968)
HumanSignal/label-studio: docs/source/guide/storage.md

### Prerequisites

> You must set two environment variables in Label Studio to connect to Azure Blob storage:

Reviewer: ```suggestion
You can use either an Account Name and Account Key, or an Account Name and SAS Token to connect to Azure Blob storage. For the connection type you choose, set two environment variables in Label Studio:
```

- `AZURE_BLOB_ACCOUNT_NAME` to specify the name of the storage account.

> - `AZURE_BLOB_ACCOUNT_KEY` to specify the secret key for the storage account, ***OR***

Reviewer: ```suggestion
- `AZURE_BLOB_ACCOUNT_KEY` to specify the secret key for the storage account, or
```

emphasis is unnecessary

> - `AZURE_BLOB_SAS_TOKEN` to specifcy the SharedAccessSignature for the storage account.

Reviewer: ```suggestion
- `AZURE_BLOB_SAS_TOKEN` to specify the SharedAccessSignature for the storage account.
```

7. Adjust the remaining optional parameters:
    - In the **File Filter Regex** field, specify a regular expression to filter bucket objects. Use `.*` to collect all objects.
    - In the **Account Name** field, specify the account name for the Azure storage. You can also set this field as an environment variable,`AZURE_BLOB_ACCOUNT_NAME`.
    - In the **Account Key** field, specify the secret key to access the storage account. You can also set this field as an environment variable,`AZURE_BLOB_ACCOUNT_KEY`. 

>     - Alternatively, if you do not have the Account Key, the **SAS Token** field specifically allows for authentication using the SharedAccessSignature for the given container. It may alsos be set as an environmental variable `AZURE_BLOB_SAS_TOKEN`.

Reviewer 2: small typo alsos -> also

>     - Alternatively, if you do not have the Account Key, the **SAS Token** field specifically allows for authentication using the SharedAccessSignature for the given container. It may alsos be set as an environmental variable `AZURE_BLOB_SAS_TOKEN`.

Reviewer: ```suggestion
    - If you do not have an Account Key, use the **SAS Token** field to authenticate to the given container using the SharedAccessSignature. You can also set this token as an environmental variable: `AZURE_BLOB_SAS_TOKEN`.
```

Reviewer: this one too please, then I'm ready to approve! Thanks!
